src/views/preview/preview.scss

```diff
@@ -32,13 +32,23 @@ $stage-width: 480px;
                 transform: translate(22rem, 0);
             }
         }
+
+        &.no-edit {
+            /* titles of projects you don't own should not
+            show the full title if this is too long */
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+        }
     }
 
     .project-header {
         margin-right: 2rem;
+        width: 65%; /* Setting a width to the header will keep it from flowing over the usual content of the page */
```

Comment:
Where does `65%` come from? I would expect this to be 100% and constrained by the parent component with margins or flex-basis or something like that?

Reply:
I have tried for two hours now to get it to work without setting the width this way but it doesn't. Flex basis seems to require a width also. Changing settings for the button doesn't change the width of that input field either. I think putting the width here is the easiest and for what I have tested safest thing we can do.

Reply:
ACTUALLY this is so interesting that I am now reading more about flex box in general and I might have found a solution. Will investigate further. Hint: min-width 0!

Reply:
OMG RAY IT WORKS!